themes/classic/templates/catalog/_partials/product-variants.tpl

```diff
@@ -35,4 +35,5 @@
       {/if}
     </div>
   {/foreach}
+  <input type="hidden" name="idCombination" id="idCombination" value="{$product.id_product_attribute}">
```

Comment:
we don't need it in this template and as you can see the value is available from templates, so what is the gain to submit this field ?

When you need the combination Id, call `$product.id_product_attribute`